compiler/luci/pass/src/QuantizedModelVerifier.test.cpp

```diff
@@ -559,90 +645,52 @@ TEST(QuantizedModelVerifierTest, LocalCreateDummyConst)
   EXPECT_NO_THROW(create_dummy_const<Type::FLOAT32>(&g, {32, 32}));
 }
 
-TEST(QuantizedModelVerifierTest, Logistic)
-{
-  TEST_WITH_GRAPH(LogisticTestGraph, Type::U8, Granularity::LayerWise);
-  TEST_WITH_GRAPH(LogisticTestGraph, Type::U8, Granularity::ChannelWise);
-  TEST_WITH_GRAPH(LogisticTestGraph, Type::S16, Granularity::ChannelWise);
-}
+TEST(QuantizedModelVerifierTest, Logistic) { test_with_graph<LogisticTestGraph>(); }
```

Comment:
Out test analyzer will fail as there is no gtest macros something like `ASSERT_XXX`

Reply:
Build and test succeeded.
Do you mean CircleCI analyzer or svace?

<details>

./build/compiler/luci/pass/luci_pass_test
Running main() from gtest_main.cc
[==========] Running 190 tests from 63 test cases.
[----------] Global test environment set-up.
[----------] 3 tests from BatchNormPatternFinderMulAddTest
[ RUN      ] BatchNormPatternFinderMulAddTest.is_batchnorm_add
[       OK ] BatchNormPatternFinderMulAddTest.is_batchnorm_add (0 ms)
[ RUN      ] BatchNormPatternFinderMulAddTest.is_batchnorm_add2
[       OK ] BatchNormPatternFinderMulAddTest.is_batchnorm_add2 (0 ms)
[ RUN      ] BatchNormPatternFinderMulAddTest.is_batchnorm_mul
[       OK ] BatchNormPatternFinderMulAddTest.is_batchnorm_mul (0 ms)
[----------] 3 tests from BatchNormPatternFinderMulAddTest (0 ms total)

[----------] 1 test from BatchNormPatternFinderAddTest
[ RUN      ] BatchNormPatternFinderAddTest.is_batchnorm_add_NEG
[       OK ] BatchNormPatternFinderAddTest.is_batchnorm_add_NEG (0 ms)
[----------] 1 test from BatchNormPatternFinderAddTest (0 ms total)

[----------] 13 tests from CircleOptimizerTest
[ RUN      ] CircleOptimizerTest.optimize_algorithms
[       OK ] CircleOptimizerTest.optimize_algorithms (1 ms)
[ RUN      ] CircleOptimizerTest.sparsify_simple
[       OK ] CircleOptimizerTest.sparsify_simple (0 ms)
[ RUN      ] CircleOptimizerTest.quantize_quantdequant_simple
[       OK ] CircleOptimizerTest.quantize_quantdequant_simple (0 ms)
[ RUN      ] CircleOptimizerTest.quantize_quantdequant_input_NEG
[       OK ] CircleOptimizerTest.quantize_quantdequant_input_NEG (0 ms)
[ RUN      ] CircleOptimizerTest.quantize_quantdequant_output_NEG
[       OK ] CircleOptimizerTest.quantize_quantdequant_output_NEG (0 ms)
[ RUN      ] CircleOptimizerTest.quantize_quantdequant_gran_NEG
[       OK ] CircleOptimizerTest.quantize_quantdequant_gran_NEG (0 ms)
[ RUN      ] CircleOptimizerTest.quantize_minmax_simple
[       OK ] CircleOptimizerTest.quantize_minmax_simple (0 ms)
[ RUN      ] CircleOptimizerTest.quantize_minmax_input_NEG
[       OK ] CircleOptimizerTest.quantize_minmax_input_NEG (0 ms)
[ RUN      ] CircleOptimizerTest.quantize_minmax_output_NEG
[       OK ] CircleOptimizerTest.quantize_minmax_output_NEG (0 ms)
[ RUN      ] CircleOptimizerTest.quantize_minmax_gran_NEG
[       OK ] CircleOptimizerTest.quantize_minmax_gran_NEG (0 ms)
[ RUN      ] CircleOptimizerTest.quantize_requant_simple
[       OK ] CircleOptimizerTest.quantize_requant_simple (0 ms)
[ RUN      ] CircleOptimizerTest.quantize_requant_input_NEG
[       OK ] CircleOptimizerTest.quantize_requant_input_NEG (0 ms)
[ RUN      ] CircleOptimizerTest.quantize_requant_output_NEG
[       OK ] CircleOptimizerTest.quantize_requant_output_NEG (0 ms)
[----------] 13 tests from CircleOptimizerTest (1 ms total)

[----------] 4 tests from CircleShapeInferencePassTest
[ RUN      ] CircleShapeInferencePassTest.name
[       OK ] CircleShapeInferencePassTest.name (0 ms)
[ RUN      ] CircleShapeInferencePassTest.original_node_change
[       OK ] CircleShapeInferencePassTest.original_node_change (1 ms)
[ RUN      ] CircleShapeInferencePassTest.wrong_imported_shape
[       OK ] CircleShapeInferencePassTest.wrong_imported_shape (0 ms)
[ RUN      ] CircleShapeInferencePassTest.not_used_virtual_op
[       OK ] CircleShapeInferencePassTest.not_used_virtual_op (0 ms)
[----------] 4 tests from CircleShapeInferencePassTest (1 ms total)

[----------] 1 test from CircleTypeInferencePassTest
[ RUN      ] CircleTypeInferencePassTest.name
[       OK ] CircleTypeInferencePassTest.name (0 ms)
[----------] 1 test from CircleTypeInferencePassTest (0 ms total)

[----------] 1 test from ConvertNCHWToNHWCPassTest
[ RUN      ] ConvertNCHWToNHWCPassTest.name
[       OK ] ConvertNCHWToNHWCPassTest.name (0 ms)
[----------] 1 test from ConvertNCHWToNHWCPassTest (0 ms total)

[----------] 10 tests from ConvertNCHWToNHWC
[ RUN      ] ConvertNCHWToNHWC.Add
[       OK ] ConvertNCHWToNHWC.Add (1 ms)
[ RUN      ] ConvertNCHWToNHWC.Concatenation
[       OK ] ConvertNCHWToNHWC.Concatenation (0 ms)
[ RUN      ] ConvertNCHWToNHWC.LeakyRelu
[       OK ] ConvertNCHWToNHWC.LeakyRelu (0 ms)
[ RUN      ] ConvertNCHWToNHWC.Mul
[       OK ] ConvertNCHWToNHWC.Mul (1 ms)
[ RUN      ] ConvertNCHWToNHWC.Neg
[       OK ] ConvertNCHWToNHWC.Neg (0 ms)
[ RUN      ] ConvertNCHWToNHWC.Pad
[       OK ] ConvertNCHWToNHWC.Pad (0 ms)
[ RUN      ] ConvertNCHWToNHWC.Unknown_Shape_NEG
[       OK ] ConvertNCHWToNHWC.Unknown_Shape_NEG (1 ms)
[ RUN      ] ConvertNCHWToNHWC.Preserve_Input_Output
[       OK ] ConvertNCHWToNHWC.Preserve_Input_Output (1 ms)
[ RUN      ] ConvertNCHWToNHWC.Relu
[       OK ] ConvertNCHWToNHWC.Relu (0 ms)
[ RUN      ] ConvertNCHWToNHWC.Relu6
[       OK ] ConvertNCHWToNHWC.Relu6 (0 ms)
[----------] 10 tests from ConvertNCHWToNHWC (4 ms total)

[----------] 1 test from FoldAddV2PassTest
[ RUN      ] FoldAddV2PassTest.name
[       OK ] FoldAddV2PassTest.name (0 ms)
[----------] 1 test from FoldAddV2PassTest (0 ms total)

[----------] 2 tests from FoldS64AddV2Test
[ RUN      ] FoldS64AddV2Test.fold_addV2
[       OK ] FoldS64AddV2Test.fold_addV2 (0 ms)
[ RUN      ] FoldS64AddV2Test.input_type_mismatch_NEG
[       OK ] FoldS64AddV2Test.input_type_mismatch_NEG (0 ms)
[----------] 2 tests from FoldS64AddV2Test (0 ms total)

[----------] 1 test from FoldCastPassTest
[ RUN      ] FoldCastPassTest.name
[       OK ] FoldCastPassTest.name (0 ms)
[----------] 1 test from FoldCastPassTest (0 ms total)

[----------] 1 test from FoldS64ToS32CastTest
[ RUN      ] FoldS64ToS32CastTest.fold_cast_s64_to_s32
[       OK ] FoldS64ToS32CastTest.fold_cast_s64_to_s32 (1 ms)
[----------] 1 test from FoldS64ToS32CastTest (1 ms total)

[----------] 1 test from FoldDequantizePassTest
[ RUN      ] FoldDequantizePassTest.name
[       OK ] FoldDequantizePassTest.name (0 ms)
[----------] 1 test from FoldDequantizePassTest (0 ms total)

[----------] 1 test from FoldSparseToDensePassTest
[ RUN      ] FoldSparseToDensePassTest.name
[       OK ] FoldSparseToDensePassTest.name (0 ms)
[----------] 1 test from FoldSparseToDensePassTest (0 ms total)

[----------] 2 tests from S64SparseToDenseZeroIndicesTest
[ RUN      ] S64SparseToDenseZeroIndicesTest.fold_stod_with_zero_indices
[       OK ] S64SparseToDenseZeroIndicesTest.fold_stod_with_zero_indices (0 ms)
[ RUN      ] S64SparseToDenseZeroIndicesTest.illegal_input_NEG
[       OK ] S64SparseToDenseZeroIndicesTest.illegal_input_NEG (0 ms)
[----------] 2 tests from S64SparseToDenseZeroIndicesTest (0 ms total)

[----------] 1 test from ForwardReshapeToUnaryOpPassTest
[ RUN      ] ForwardReshapeToUnaryOpPassTest.name
[       OK ] ForwardReshapeToUnaryOpPassTest.name (0 ms)
[----------] 1 test from ForwardReshapeToUnaryOpPassTest (0 ms total)

[----------] 1 test from ForwardReshapeToNegGraphTest
[ RUN      ] ForwardReshapeToNegGraphTest.simple_forward
[       OK ] ForwardReshapeToNegGraphTest.simple_forward (0 ms)
[----------] 1 test from ForwardReshapeToNegGraphTest (0 ms total)

[----------] 1 test from FuseActivationFunctionPassTest
[ RUN      ] FuseActivationFunctionPassTest.name
[       OK ] FuseActivationFunctionPassTest.name (0 ms)
[----------] 1 test from FuseActivationFunctionPassTest (0 ms total)

[----------] 12 tests from FusePreActivationBatchNorm
[ RUN      ] FusePreActivationBatchNorm.fuse_activation_function
[       OK ] FusePreActivationBatchNorm.fuse_activation_function (0 ms)
[ RUN      ] FusePreActivationBatchNorm.fuse_activation_function_dup_relu
[       OK ] FusePreActivationBatchNorm.fuse_activation_function_dup_relu (0 ms)
[ RUN      ] FusePreActivationBatchNorm.fuse_activation_function_mulsucc_NEG
[       OK ] FusePreActivationBatchNorm.fuse_activation_function_mulsucc_NEG (0 ms)
[ RUN      ] FusePreActivationBatchNorm.fuse_activation_function_tanh_NEG
[       OK ] FusePreActivationBatchNorm.fuse_activation_function_tanh_NEG (0 ms)
[ RUN      ] FusePreActivationBatchNorm.swap_mul_add
[       OK ] FusePreActivationBatchNorm.swap_mul_add (0 ms)
[ RUN      ] FusePreActivationBatchNorm.swap_mul_add_NEG
[       OK ] FusePreActivationBatchNorm.swap_mul_add_NEG (0 ms)
[ RUN      ] FusePreActivationBatchNorm.fuse_mul_with_conv
[       OK ] FusePreActivationBatchNorm.fuse_mul_with_conv (0 ms)
[ RUN      ] FusePreActivationBatchNorm.fuse_mul_with_conv_NEG
[       OK ] FusePreActivationBatchNorm.fuse_mul_with_conv_NEG (0 ms)
[ RUN      ] FusePreActivationBatchNorm.fuse_add_with_conv
[       OK ] FusePreActivationBatchNorm.fuse_add_with_conv (1 ms)
[ RUN      ] FusePreActivationBatchNorm.fuse_add_with_conv_NEG
[       OK ] FusePreActivationBatchNorm.fuse_add_with_conv_NEG (0 ms)
[ RUN      ] FusePreActivationBatchNorm.fuse_sub_with_conv
[       OK ] FusePreActivationBatchNorm.fuse_sub_with_conv (0 ms)
[ RUN      ] FusePreActivationBatchNorm.fuse_sub_with_conv_NEG
[       OK ] FusePreActivationBatchNorm.fuse_sub_with_conv_NEG (0 ms)
[----------] 12 tests from FusePreActivationBatchNorm (1 ms total)

[----------] 1 test from FuseAddWithTConvPassTest
[ RUN      ] FuseAddWithTConvPassTest.name
[       OK ] FuseAddWithTConvPassTest.name (0 ms)
[----------] 1 test from FuseAddWithTConvPassTest (0 ms total)

[----------] 1 test from FuseBCQPassTest
[ RUN      ] FuseBCQPassTest.name
[       OK ] FuseBCQPassTest.name (0 ms)
[----------] 1 test from FuseBCQPassTest (0 ms total)

[----------] 1 test from FuseBatchNormWithConvPassTest
[ RUN      ] FuseBatchNormWithConvPassTest.name
[       OK ] FuseBatchNormWithConvPassTest.name (0 ms)
[----------] 1 test from FuseBatchNormWithConvPassTest (0 ms total)

[----------] 1 test from FuseBatchNormWithDwConvPassTest
[ RUN      ] FuseBatchNormWithDwConvPassTest.name
[       OK ] FuseBatchNormWithDwConvPassTest.name (0 ms)
[----------] 1 test from FuseBatchNormWithDwConvPassTest (0 ms total)

[----------] 1 test from FuseBatchNormWithTConvPassTest
[ RUN      ] FuseBatchNormWithTConvPassTest.name
[       OK ] FuseBatchNormWithTConvPassTest.name (0 ms)
[----------] 1 test from FuseBatchNormWithTConvPassTest (0 ms total)

[----------] 1 test from FuseInstanceNormPassTest
[ RUN      ] FuseInstanceNormPassTest.name
[       OK ] FuseInstanceNormPassTest.name (0 ms)
[----------] 1 test from FuseInstanceNormPassTest (0 ms total)

[----------] 1 test from FuseInstanceNormPass
[ RUN      ] FuseInstanceNormPass.is_quasi_1D_with_dummy_dim
[       OK ] FuseInstanceNormPass.is_quasi_1D_with_dummy_dim (0 ms)
[----------] 1 test from FuseInstanceNormPass (0 ms total)

[----------] 1 test from FusePreActivationBatchNormPassTest
[ RUN      ] FusePreActivationBatchNormPassTest.name
[       OK ] FusePreActivationBatchNormPassTest.name (0 ms)
[----------] 1 test from FusePreActivationBatchNormPassTest (0 ms total)

[----------] 1 test from MakeBatchNormGammaPositivePassTest
[ RUN      ] MakeBatchNormGammaPositivePassTest.name
[       OK ] MakeBatchNormGammaPositivePassTest.name (0 ms)
[----------] 1 test from MakeBatchNormGammaPositivePassTest (0 ms total)

[----------] 2 tests from ModulePhaseTest
[ RUN      ] ModulePhaseTest.saturate
[       OK ] ModulePhaseTest.saturate (0 ms)
[ RUN      ] ModulePhaseTest.restart
[       OK ] ModulePhaseTest.restart (0 ms)
[----------] 2 tests from ModulePhaseTest (0 ms total)

[----------] 4 tests from PropagateConcatenationQparam
[ RUN      ] PropagateConcatenationQparam.propagate_concat_quantparam_u8
[       OK ] PropagateConcatenationQparam.propagate_concat_quantparam_u8 (0 ms)
[ RUN      ] PropagateConcatenationQparam.propagate_concat_quantparam_u8_NEG
[       OK ] PropagateConcatenationQparam.propagate_concat_quantparam_u8_NEG (0 ms)
[ RUN      ] PropagateConcatenationQparam.propagate_concat_quantparam_i16
[       OK ] PropagateConcatenationQparam.propagate_concat_quantparam_i16 (0 ms)
[ RUN      ] PropagateConcatenationQparam.propagate_concat_quantparam_i16_NEG
[       OK ] PropagateConcatenationQparam.propagate_concat_quantparam_i16_NEG (0 ms)
[----------] 4 tests from PropagateConcatenationQparam (0 ms total)

[----------] 1 test from PropagateQuantParamPassTest
[ RUN      ] PropagateQuantParamPassTest.name
[       OK ] PropagateQuantParamPassTest.name (0 ms)
[----------] 1 test from PropagateQuantParamPassTest (0 ms total)

[----------] 2 tests from PropagateQuantParam
[ RUN      ] PropagateQuantParam.simple
[       OK ] PropagateQuantParam.simple (0 ms)
[ RUN      ] PropagateQuantParam.wrong_op_NEG
[       OK ] PropagateQuantParam.wrong_op_NEG (0 ms)
[----------] 2 tests from PropagateQuantParam (0 ms total)

[----------] 1 test from QuantizeDequantizeWeightsPassTest
[ RUN      ] QuantizeDequantizeWeightsPassTest.name
[       OK ] QuantizeDequantizeWeightsPassTest.name (0 ms)
[----------] 1 test from QuantizeDequantizeWeightsPassTest (0 ms total)

[----------] 1 test from QuantizeWithMinMaxPassTest
[ RUN      ] QuantizeWithMinMaxPassTest.name
[       OK ] QuantizeWithMinMaxPassTest.name (0 ms)
[----------] 1 test from QuantizeWithMinMaxPassTest (0 ms total)

[----------] 48 tests from QuantizedModelVerifierTest
[ RUN      ] QuantizedModelVerifierTest.LocalCreateDummyConst
[       OK ] QuantizedModelVerifierTest.LocalCreateDummyConst (0 ms)
[ RUN      ] QuantizedModelVerifierTest.Logistic
[       OK ] QuantizedModelVerifierTest.Logistic (1 ms)
[ RUN      ] QuantizedModelVerifierTest.Logistic_wrong_type_NEG
[       OK ] QuantizedModelVerifierTest.Logistic_wrong_type_NEG (0 ms)
[ RUN      ] QuantizedModelVerifierTest.Logistic_wrong_granularity_NEG
[       OK ] QuantizedModelVerifierTest.Logistic_wrong_granularity_NEG (0 ms)
[ RUN      ] QuantizedModelVerifierTest.Softmax
[       OK ] QuantizedModelVerifierTest.Softmax (1 ms)
[ RUN      ] QuantizedModelVerifierTest.Softmax_wrong_type_NEG
[       OK ] QuantizedModelVerifierTest.Softmax_wrong_type_NEG (0 ms)
[ RUN      ] QuantizedModelVerifierTest.Softmax_wrong_granularity_NEG
[       OK ] QuantizedModelVerifierTest.Softmax_wrong_granularity_NEG (0 ms)
[ RUN      ] QuantizedModelVerifierTest.Slice
[       OK ] QuantizedModelVerifierTest.Slice (1 ms)
[ RUN      ] QuantizedModelVerifierTest.Slice_wrong_type_NEG
[       OK ] QuantizedModelVerifierTest.Slice_wrong_type_NEG (0 ms)
[ RUN      ] QuantizedModelVerifierTest.Slice_wrong_granularity_NEG
[       OK ] QuantizedModelVerifierTest.Slice_wrong_granularity_NEG (1 ms)
[ RUN      ] QuantizedModelVerifierTest.ArgMax
[       OK ] QuantizedModelVerifierTest.ArgMax (1 ms)
[ RUN      ] QuantizedModelVerifierTest.ArgMax_wrong_dimension_type_NEG
[       OK ] QuantizedModelVerifierTest.ArgMax_wrong_dimension_type_NEG (0 ms)
[ RUN      ] QuantizedModelVerifierTest.ArgMax_wrong_input_granularity_NEG
[       OK ] QuantizedModelVerifierTest.ArgMax_wrong_input_granularity_NEG (0 ms)
[ RUN      ] QuantizedModelVerifierTest.Concatenation
[       OK ] QuantizedModelVerifierTest.Concatenation (0 ms)
[ RUN      ] QuantizedModelVerifierTest.Concatenation_wrong_type_NEG
[       OK ] QuantizedModelVerifierTest.Concatenation_wrong_type_NEG (1 ms)
[ RUN      ] QuantizedModelVerifierTest.Concatenation_wrong_granularity_NEG
[       OK ] QuantizedModelVerifierTest.Concatenation_wrong_granularity_NEG (0 ms)
[ RUN      ] QuantizedModelVerifierTest.LogicalOr
[       OK ] QuantizedModelVerifierTest.LogicalOr (0 ms)
[ RUN      ] QuantizedModelVerifierTest.LogicalOr_wrong_type_NEG
[       OK ] QuantizedModelVerifierTest.LogicalOr_wrong_type_NEG (1 ms)
[ RUN      ] QuantizedModelVerifierTest.Reshape
[       OK ] QuantizedModelVerifierTest.Reshape (0 ms)
[ RUN      ] QuantizedModelVerifierTest.Reshape_wrong_type_NEG
[       OK ] QuantizedModelVerifierTest.Reshape_wrong_type_NEG (0 ms)
[ RUN      ] QuantizedModelVerifierTest.Reshape_wrong_granularity_NEG
[       OK ] QuantizedModelVerifierTest.Reshape_wrong_granularity_NEG (1 ms)
[ RUN      ] QuantizedModelVerifierTest.Tanh
[       OK ] QuantizedModelVerifierTest.Tanh (0 ms)
[ RUN      ] QuantizedModelVerifierTest.Tanh_wrong_type_NEG
[       OK ] QuantizedModelVerifierTest.Tanh_wrong_type_NEG (0 ms)
[ RUN      ] QuantizedModelVerifierTest.Tanh_wrong_granularity_NEG
[       OK ] QuantizedModelVerifierTest.Tanh_wrong_granularity_NEG (0 ms)
[ RUN      ] QuantizedModelVerifierTest.Pad
[       OK ] QuantizedModelVerifierTest.Pad (1 ms)
[ RUN      ] QuantizedModelVerifierTest.Pad_wrong_type_NEG
[       OK ] QuantizedModelVerifierTest.Pad_wrong_type_NEG (0 ms)
[ RUN      ] QuantizedModelVerifierTest.Pad_wrong_granularity_NEG
[       OK ] QuantizedModelVerifierTest.Pad_wrong_granularity_NEG (0 ms)
[ RUN      ] QuantizedModelVerifierTest.Transpose
[       OK ] QuantizedModelVerifierTest.Transpose (1 ms)
[ RUN      ] QuantizedModelVerifierTest.Transpose_wrong_type_NEG
[       OK ] QuantizedModelVerifierTest.Transpose_wrong_type_NEG (0 ms)
[ RUN      ] QuantizedModelVerifierTest.Transpose_wrong_granularity_NEG
[       OK ] QuantizedModelVerifierTest.Transpose_wrong_granularity_NEG (0 ms)
[ RUN      ] QuantizedModelVerifierTest.Floor
[       OK ] QuantizedModelVerifierTest.Floor (0 ms)
[ RUN      ] QuantizedModelVerifierTest.Floor_wrong_type_NEG
[       OK ] QuantizedModelVerifierTest.Floor_wrong_type_NEG (1 ms)
[ RUN      ] QuantizedModelVerifierTest.Floor_wrong_granularity_NEG
[       OK ] QuantizedModelVerifierTest.Floor_wrong_granularity_NEG (0 ms)
[ RUN      ] QuantizedModelVerifierTest.GreaterEqual
[       OK ] QuantizedModelVerifierTest.GreaterEqual (0 ms)
[ RUN      ] QuantizedModelVerifierTest.GreaterEqual_wrong_type_NEG
[       OK ] QuantizedModelVerifierTest.GreaterEqual_wrong_type_NEG (1 ms)
[ RUN      ] QuantizedModelVerifierTest.GreaterEqual_wrong_granularity_NEG
[       OK ] QuantizedModelVerifierTest.GreaterEqual_wrong_granularity_NEG (0 ms)
[ RUN      ] QuantizedModelVerifierTest.Greater
[       OK ] QuantizedModelVerifierTest.Greater (1 ms)
[ RUN      ] QuantizedModelVerifierTest.Greater_wrong_type_NEG
[       OK ] QuantizedModelVerifierTest.Greater_wrong_type_NEG (0 ms)
[ RUN      ] QuantizedModelVerifierTest.Greater_wrong_granularity_NEG
[       OK ] QuantizedModelVerifierTest.Greater_wrong_granularity_NEG (1 ms)
[ RUN      ] QuantizedModelVerifierTest.NotEqual
[       OK ] QuantizedModelVerifierTest.NotEqual (0 ms)
[ RUN      ] QuantizedModelVerifierTest.NotEqual_wrong_type_NEG
[       OK ] QuantizedModelVerifierTest.NotEqual_wrong_type_NEG (0 ms)
[ RUN      ] QuantizedModelVerifierTest.NotEqual_wrong_granularity_NEG
[       OK ] QuantizedModelVerifierTest.NotEqual_wrong_granularity_NEG (1 ms)
[ RUN      ] QuantizedModelVerifierTest.Div
[       OK ] QuantizedModelVerifierTest.Div (0 ms)
[ RUN      ] QuantizedModelVerifierTest.Div_wrong_type_NEG
[       OK ] QuantizedModelVerifierTest.Div_wrong_type_NEG (1 ms)
[ RUN      ] QuantizedModelVerifierTest.Div_wrong_granularity_NEG
[       OK ] QuantizedModelVerifierTest.Div_wrong_granularity_NEG (0 ms)
[ RUN      ] QuantizedModelVerifierTest.FloorDiv
[       OK ] QuantizedModelVerifierTest.FloorDiv (1 ms)
[ RUN      ] QuantizedModelVerifierTest.FloorDiv_wrong_type_NEG
[       OK ] QuantizedModelVerifierTest.FloorDiv_wrong_type_NEG (0 ms)
[ RUN      ] QuantizedModelVerifierTest.FloorDiv_wrong_granularity_NEG
[       OK ] QuantizedModelVerifierTest.FloorDiv_wrong_granularity_NEG (1 ms)
[----------] 48 tests from QuantizedModelVerifierTest (18 ms total)

[----------] 1 test from RemoveRedundantReshapePassTest
[ RUN      ] RemoveRedundantReshapePassTest.name
[       OK ] RemoveRedundantReshapePassTest.name (0 ms)
[----------] 1 test from RemoveRedundantReshapePassTest (0 ms total)

[----------] 1 test from RemoveRedundantReshape
[ RUN      ] RemoveRedundantReshape.simple_case
[       OK ] RemoveRedundantReshape.simple_case (0 ms)
[----------] 1 test from RemoveRedundantReshape (0 ms total)

[----------] 1 test from RemoveRedundantTransposePassTest
[ RUN      ] RemoveRedundantTransposePassTest.name
[       OK ] RemoveRedundantTransposePassTest.name (0 ms)
[----------] 1 test from RemoveRedundantTransposePassTest (0 ms total)

[----------] 4 tests from RemoveRedundantTransposePass
[ RUN      ] RemoveRedundantTransposePass.remove_consecutive_transpose_function_type1
[       OK ] RemoveRedundantTransposePass.remove_consecutive_transpose_function_type1 (0 ms)
[ RUN      ] RemoveRedundantTransposePass.remove_consecutive_transpose_function_type2
[       OK ] RemoveRedundantTransposePass.remove_consecutive_transpose_function_type2 (0 ms)
[ RUN      ] RemoveRedundantTransposePass.remove_consecutive_transpose_function_with_branch_remove_case
[       OK ] RemoveRedundantTransposePass.remove_consecutive_transpose_function_with_branch_remove_case (0 ms)
[ RUN      ] RemoveRedundantTransposePass.remove_consecutive_transpose_function_with_branch_leave_one
[       OK ] RemoveRedundantTransposePass.remove_consecutive_transpose_function_with_branch_leave_one (0 ms)
[----------] 4 tests from RemoveRedundantTransposePass (0 ms total)

[----------] 1 test from RemoveUnnecessaryReshapePassTest
[ RUN      ] RemoveUnnecessaryReshapePassTest.name
[       OK ] RemoveUnnecessaryReshapePassTest.name (0 ms)
[----------] 1 test from RemoveUnnecessaryReshapePassTest (0 ms total)

[----------] 2 tests from RemoveUnnecessaryReshapePass
[ RUN      ] RemoveUnnecessaryReshapePass.removed
[       OK ] RemoveUnnecessaryReshapePass.removed (0 ms)
[ RUN      ] RemoveUnnecessaryReshapePass.not_removed_NEG
[       OK ] RemoveUnnecessaryReshapePass.not_removed_NEG (0 ms)
[----------] 2 tests from RemoveUnnecessaryReshapePass (0 ms total)

[----------] 3 tests from RemoveUnnecessarySlicePass
[ RUN      ] RemoveUnnecessarySlicePass.name
[       OK ] RemoveUnnecessarySlicePass.name (0 ms)
[ RUN      ] RemoveUnnecessarySlicePass.removed
[       OK ] RemoveUnnecessarySlicePass.removed (0 ms)
[ RUN      ] RemoveUnnecessarySlicePass.not_removed_NEG
[       OK ] RemoveUnnecessarySlicePass.not_removed_NEG (0 ms)
[----------] 3 tests from RemoveUnnecessarySlicePass (0 ms total)

[----------] 3 tests from RemoveUnnecessarySplitPass
[ RUN      ] RemoveUnnecessarySplitPass.name
[       OK ] RemoveUnnecessarySplitPass.name (0 ms)
[ RUN      ] RemoveUnnecessarySplitPass.create_unnecessary_split
[       OK ] RemoveUnnecessarySplitPass.create_unnecessary_split (0 ms)
[ RUN      ] RemoveUnnecessarySplitPass.create_unnecessary_split_NEG
[       OK ] RemoveUnnecessarySplitPass.create_unnecessary_split_NEG (0 ms)
[----------] 3 tests from RemoveUnnecessarySplitPass (0 ms total)

[----------] 2 tests from RemoveUnnecessaryStridedSlicePass
[ RUN      ] RemoveUnnecessaryStridedSlicePass.basic_case
[       OK ] RemoveUnnecessaryStridedSlicePass.basic_case (1 ms)
[ RUN      ] RemoveUnnecessaryStridedSlicePass.basic_fail_case_NEG
[       OK ] RemoveUnnecessaryStridedSlicePass.basic_fail_case_NEG (0 ms)
[----------] 2 tests from RemoveUnnecessaryStridedSlicePass (1 ms total)

[----------] 3 tests from ReplaceMulAddWithDepthwiseConv
[ RUN      ] ReplaceMulAddWithDepthwiseConv.name
[       OK ] ReplaceMulAddWithDepthwiseConv.name (0 ms)
[ RUN      ] ReplaceMulAddWithDepthwiseConv.simple
[       OK ] ReplaceMulAddWithDepthwiseConv.simple (0 ms)
[ RUN      ] ReplaceMulAddWithDepthwiseConv.wrong_op_NEG
[       OK ] ReplaceMulAddWithDepthwiseConv.wrong_op_NEG (0 ms)
[----------] 3 tests from ReplaceMulAddWithDepthwiseConv (0 ms total)

[----------] 1 test from RequantizePassTest
[ RUN      ] RequantizePassTest.name
[       OK ] RequantizePassTest.name (0 ms)
[----------] 1 test from RequantizePassTest (0 ms total)

[----------] 1 test from ResolveCustomOpAddPassTest
[ RUN      ] ResolveCustomOpAddPassTest.name
[       OK ] ResolveCustomOpAddPassTest.name (0 ms)
[----------] 1 test from ResolveCustomOpAddPassTest (0 ms total)

[----------] 1 test from ResolveCustomOpBatchMatMulPassTest
[ RUN      ] ResolveCustomOpBatchMatMulPassTest.name
[       OK ] ResolveCustomOpBatchMatMulPassTest.name (0 ms)
[----------] 1 test from ResolveCustomOpBatchMatMulPassTest (0 ms total)

[----------] 2 tests from BatchMatmulV2GraphTest
[ RUN      ] BatchMatmulV2GraphTest.simple_test
[       OK ] BatchMatmulV2GraphTest.simple_test (0 ms)
[ RUN      ] BatchMatmulV2GraphTest.wrong_condition_NEG
[       OK ] BatchMatmulV2GraphTest.wrong_condition_NEG (0 ms)
[----------] 2 tests from BatchMatmulV2GraphTest (0 ms total)

[----------] 1 test from ResolveCustomOpMatMulPassTest
[ RUN      ] ResolveCustomOpMatMulPassTest.name
[       OK ] ResolveCustomOpMatMulPassTest.name (0 ms)
[----------] 1 test from ResolveCustomOpMatMulPassTest (0 ms total)

[----------] 4 tests from ShuffleWeightTo16x1Float32PassTest
[ RUN      ] ShuffleWeightTo16x1Float32PassTest.name
[       OK ] ShuffleWeightTo16x1Float32PassTest.name (0 ms)
[ RUN      ] ShuffleWeightTo16x1Float32PassTest.SimpleTest1
[       OK ] ShuffleWeightTo16x1Float32PassTest.SimpleTest1 (0 ms)
[ RUN      ] ShuffleWeightTo16x1Float32PassTest.invalid_weight_shape_NEG
[       OK ] ShuffleWeightTo16x1Float32PassTest.invalid_weight_shape_NEG (0 ms)
[ RUN      ] ShuffleWeightTo16x1Float32PassTest.invalid_weight_row16_NEG
[       OK ] ShuffleWeightTo16x1Float32PassTest.invalid_weight_row16_NEG (0 ms)
[----------] 4 tests from ShuffleWeightTo16x1Float32PassTest (0 ms total)

[----------] 7 tests from SparsifierTest
[ RUN      ] SparsifierTest.NoBlockDenseDense
[       OK ] SparsifierTest.NoBlockDenseDense (0 ms)
[ RUN      ] SparsifierTest.NoBlockDenseCSR
[       OK ] SparsifierTest.NoBlockDenseCSR (0 ms)
[ RUN      ] SparsifierTest.BlockDenseDense
[       OK ] SparsifierTest.BlockDenseDense (0 ms)
[ RUN      ] SparsifierTest.BlockDenseSparse
[       OK ] SparsifierTest.BlockDenseSparse (0 ms)
[ RUN      ] SparsifierTest.BlockDenseSparse_2
[       OK ] SparsifierTest.BlockDenseSparse_2 (0 ms)
[ RUN      ] SparsifierTest.WrongTraversalOrderRank_NEG
[       OK ] SparsifierTest.WrongTraversalOrderRank_NEG (0 ms)
[ RUN      ] SparsifierTest.WrongFormatRank_NEG
[       OK ] SparsifierTest.WrongFormatRank_NEG (0 ms)
[----------] 7 tests from SparsifierTest (0 ms total)

[----------] 1 test from SparsifyTensorPassTest
[ RUN      ] SparsifyTensorPassTest.name
[       OK ] SparsifyTensorPassTest.name (0 ms)
[----------] 1 test from SparsifyTensorPassTest (0 ms total)

[----------] 1 test from SubstitutePackToReshapePassTest
[ RUN      ] SubstitutePackToReshapePassTest.name
[       OK ] SubstitutePackToReshapePassTest.name (0 ms)
[----------] 1 test from SubstitutePackToReshapePassTest (1 ms total)

[----------] 2 tests from SubstitutePackToReshapePass
[ RUN      ] SubstitutePackToReshapePass.simple_case
[       OK ] SubstitutePackToReshapePass.simple_case (0 ms)
[ RUN      ] SubstitutePackToReshapePass.simple_case_neg_axis
[       OK ] SubstitutePackToReshapePass.simple_case_neg_axis (0 ms)
[----------] 2 tests from SubstitutePackToReshapePass (0 ms total)

[----------] 1 test from SubstituteSqueezeToReshapePassTest
[ RUN      ] SubstituteSqueezeToReshapePassTest.name
[       OK ] SubstituteSqueezeToReshapePassTest.name (0 ms)
[----------] 1 test from SubstituteSqueezeToReshapePassTest (0 ms total)

[----------] 6 tests from SubstituteSqueezeToReshapeTest
[ RUN      ] SubstituteSqueezeToReshapeTest.simple_with_squeeze_dims
[       OK ] SubstituteSqueezeToReshapeTest.simple_with_squeeze_dims (0 ms)
[ RUN      ] SubstituteSqueezeToReshapeTest.simple_without_squeeze_dims
[       OK ] SubstituteSqueezeToReshapeTest.simple_without_squeeze_dims (0 ms)
[ RUN      ] SubstituteSqueezeToReshapeTest.input_with_0_dims
[       OK ] SubstituteSqueezeToReshapeTest.input_with_0_dims (0 ms)
[ RUN      ] SubstituteSqueezeToReshapeTest.nothing_to_squeeze
[       OK ] SubstituteSqueezeToReshapeTest.nothing_to_squeeze (0 ms)
[ RUN      ] SubstituteSqueezeToReshapeTest.all_to_squeeze
[       OK ] SubstituteSqueezeToReshapeTest.all_to_squeeze (1 ms)
[ RUN      ] SubstituteSqueezeToReshapeTest.wrong_squeeze_dims_NEG
[       OK ] SubstituteSqueezeToReshapeTest.wrong_squeeze_dims_NEG (0 ms)
[----------] 6 tests from SubstituteSqueezeToReshapeTest (1 ms total)

[----------] 1 test from SubstituteTransposeToReshapePassTest
[ RUN      ] SubstituteTransposeToReshapePassTest.name
[       OK ] SubstituteTransposeToReshapePassTest.name (0 ms)
[----------] 1 test from SubstituteTransposeToReshapePassTest (0 ms total)

[----------] 2 tests from SubstituteTransposeToReshapeTest
[ RUN      ] SubstituteTransposeToReshapeTest.simple_case
[       OK ] SubstituteTransposeToReshapeTest.simple_case (0 ms)
[ RUN      ] SubstituteTransposeToReshapeTest.failed_to_substitute_NEG
[       OK ] SubstituteTransposeToReshapeTest.failed_to_substitute_NEG (0 ms)
[----------] 2 tests from SubstituteTransposeToReshapeTest (0 ms total)

[----------] 3 tests from TransformMinMaxToRelu6PassTest
[ RUN      ] TransformMinMaxToRelu6PassTest.name
[       OK ] TransformMinMaxToRelu6PassTest.name (0 ms)
[ RUN      ] TransformMinMaxToRelu6PassTest.simple_test
[       OK ] TransformMinMaxToRelu6PassTest.simple_test (0 ms)
[ RUN      ] TransformMinMaxToRelu6PassTest.wrong_condition_NEG
[       OK ] TransformMinMaxToRelu6PassTest.wrong_condition_NEG (0 ms)
[----------] 3 tests from TransformMinMaxToRelu6PassTest (0 ms total)

[----------] 2 tests from LuciPassHelpersInferenceCandidates
[ RUN      ] LuciPassHelpersInferenceCandidates.inference_candidates
[       OK ] LuciPassHelpersInferenceCandidates.inference_candidates (0 ms)
[ RUN      ] LuciPassHelpersInferenceCandidates.inference_candidates_NEG
[       OK ] LuciPassHelpersInferenceCandidates.inference_candidates_NEG (0 ms)
[----------] 2 tests from LuciPassHelpersInferenceCandidates (0 ms total)

[----------] 2 tests from NodeFillerTest
[ RUN      ] NodeFillerTest.simple_test
[       OK ] NodeFillerTest.simple_test (0 ms)
[ RUN      ] NodeFillerTest.wrong_condition_NEG
[       OK ] NodeFillerTest.wrong_condition_NEG (0 ms)
[----------] 2 tests from NodeFillerTest (0 ms total)

[----------] 3 tests from StringsTest
[ RUN      ] StringsTest.str_to_dtype
[       OK ] StringsTest.str_to_dtype (0 ms)
[ RUN      ] StringsTest.str_to_granularity
[       OK ] StringsTest.str_to_granularity (0 ms)
[ RUN      ] StringsTest.csv_to_vector_int32
[       OK ] StringsTest.csv_to_vector_int32 (0 ms)
[----------] 3 tests from StringsTest (0 ms total)

[----------] 3 tests from TypeMapperTest
[ RUN      ] TypeMapperTest.simple_test
[       OK ] TypeMapperTest.simple_test (0 ms)
[ RUN      ] TypeMapperTest.with_template_test
[       OK ] TypeMapperTest.with_template_test (0 ms)
[ RUN      ] TypeMapperTest.wrong_condition_NEG
[       OK ] TypeMapperTest.wrong_condition_NEG (0 ms)
[----------] 3 tests from TypeMapperTest (0 ms total)

[----------] Global test environment tear-down
[==========] 190 tests from 63 test cases ran. (29 ms total)
[  PASSED  ] 190 tests.


</details>

Reply:
> Do you mean CircleCI analyzer or svace?

Not sure, but maybe @llFreetimell may know...

Reply:
I don't know neither unless I run the analysis....
Unfortunately, analysis cannot be done for this PR because it is not in master branch...

Reply:
@llFreetimell , is current master OK? `TEST_WITH_GRAPH` macro has `ASSERT_XXX` but test function doesn't....

Reply:
@seanshpark Oh, it was a long time ago I ran the analysis..! I started to run the analysis now and it will take about 2 hours O.O
I will let you know when the analysis is finished :)

Reply:
Thank you! :)

Reply:
@llFreetimell Thank you for the reply!! :-D

Reply:
Static analyzer does not report anything related with GTest :)

Reply:
By help from @lemmaa , found out that we need to run `infra/scripts/build-tcm.sh` (Service is in `.ahub` but not sure how to access this...). I'm currenrly running in my local...
- `tca-standalone-0.0.8.jar` is now `tca-standalone-1.0.0.jar`

Reply:
Found out that current code has issue with TCM: #6571

Reply:
@seanshpark Thank you !! 😄

Reply:
So the current QuantizedModelVerifier.test.cpp has the same issue.

Reply:
we may change all the tests in another context

Reply:
I will split the changes into smaller prs. 😄